src/node_platform.h

```diff
@@ -145,9 +145,14 @@ class NodePlatform : public MultiIsolatePlatform {
   void CallOnWorkerThread(std::unique_ptr<v8::Task> task) override;
   void CallDelayedOnWorkerThread(std::unique_ptr<v8::Task> task,
                                  double delay_in_seconds) override;
-  void CallOnForegroundThread(v8::Isolate* isolate, v8::Task* task) override;
-  void CallDelayedOnForegroundThread(v8::Isolate* isolate, v8::Task* task,
-                                     double delay_in_seconds) override;
+  void CallOnForegroundThread(v8::Isolate* isolate, v8::Task* task) override {
+    abort();
```

Comment:
can these be `UNREACHABLE()`?

Reply:
IIUC this is copying V8's
https://github.com/nodejs/node/blob/ff8539e9e747db61a2adac6d4498ef4ee5826900/deps/v8/include/v8-platform.h#L396-L400
but our `UNREACHABLE()` or even `ABORT()` have more specific semantics.